incorporating-accessibility-into-your-dev-process/demo/src/ConsistentDesign/ConsistentDesign.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+const ConsistentDesign = () => {
+  return (
+    <section>
+      <h2>Consistent design</h2>
+      <p>Folks at Bitovi have expertise in the following areas:</p>
+      <ul>
+        <li><a href="https://www.bitovi.com/services/agile-project-management-consulting">Project Management</a></li>
```

Comment:
Do you think its worth adding react router to the project to use their `Link` component? most react linking is tied to the routing system

Reply:
Definitely not for this example because it links to another page. Do people use `Link` to link to other websites? If so, I’m curious why because an `<a>` element is so much more straightforward.

Reply:
Other websites no. But other pages in the same website yes.

Reply:
I meant in general since were talking about linking and buttons

Reply:
Ah yeah, I’m probably going to cover focus management and assuming I do, I’ll build an exercise that uses `react-router-dom`.